release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -0,0 +1,1627 @@
+
+.Monitoring Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.17 |4.18 |4.19
+
+|Metrics Collection Profiles
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|====
+
+[discrete]
+[id="ocp-release-notes-web-console-tech-preview_{context}"]
+=== Web console Technology Preview features
+
+.Web console Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.17 |4.18 |4.19
+
+|{ols-official} in the {product-title} web console
+|Technology Preview
+|Technology Preview
+|Technology Peview
+|====
+
+[discrete]
+[id="ocp-release-notes-multi-arch-tech-preview_{context}"]
+=== Multi-Architecture Technology Preview features
+
+.Multi-Architecture Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.17 |4.18 |4.19
+
+|`kdump` on `arm64` architecture
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|`kdump` on `s390x` architecture
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|`kdump` on `ppc64le` architecture
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Multiarch Tuning Operator
+|General Availability
+|General Availability
+|General Availability
+
+|Support for configuring the image stream import mode behavior
+|Not Available
+|Technology Preview
+|Technology Preview
+|====
+
+[discrete]
+[id="ocp-release-notes-networking-tech-preview_{context}"]
+=== Networking Technology Preview features
+
+.Networking Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.17 |4.18 |4.19
+
+|eBPF manager Operator
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Advertise using L2 mode the MetalLB service from a subset of nodes, using a specific pool of IP addresses
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Updating the interface-specific safe sysctls list
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Egress service custom resource
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|VRF specification in `BGPPeer` custom resource
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|VRF specification in `NodeNetworkConfigurationPolicy` custom resource
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Host network settings for SR-IOV VFs
+|General Availability
+|General Availability
+|General Availability
+
+|Integration of MetalLB and FRR-K8s
+|General Availability
+|General Availability
+|General Availability
+
+|Automatic leap seconds handling for PTP grandmaster clocks
+|General Availability
+|General Availability
+|General Availability
+
+|PTP events REST API v2
+|General Availability
+|General Availability
+|General Availability
+
+|Customized `br-ex` bridge needed by OVN-Kuberenetes to use NMState
+|Technology Preview
+|General Availability
+|General Availability
+
+| Live migration to OVN-Kubernetes from OpenShift SDN
+|General Availability
+|Not Available
+|Not Available
```

Comment:
This looks more like a removed feature than TP. Might be good to have someone in this area look at this one for placement

Reply:
Pinged Joe for clarity.

Reply:
This is sort of a weird nwt thing. The TL;DR is this is correct. cc @stevsmit